Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md

```diff
@@ -0,0 +1,35 @@
+
+Fetch indicators from a JSON feed. The integration allows a great amount of user configuration to support different types of JSON feeds.
+
+* **URL** - URL of the feed.
+* **Indicator Type** - The type of indicators in the feed.
+* **Username + Password** - Credentials to access feeds that require basic authentication. 
+These fields also support the use of API key headers. To use API key headers, specify the header name and value in the following format:
+`_header:<header_name>` in the **Username** field and the header value in the **Password** field.
+* **JMESPath Extractor** - JMESPath expression for extracting the indicators from. You can check the expression in 
+the [JMESPath site](http://jmespath.org/) to verify this expression will return the following array of objects.
+* **JSON Attribute Indicator** - JSON attribute to use as indicator. Default is 'indicator'.
+* **Field Names** - The names to apply the fields in the JSON feed.
+
+## Step by step configuration
+As an example, we'll be looking at the IP ranges from Amazon AWS. This feed will ingest indicators of type CIDR. These are the feed instance configuration parameters for our example.
+
+**URL**: https://ip-ranges.amazonaws.com/ip-ranges.json.
```

Comment:
If you are using plaintext as the example values for all of these configuration step parameters - then please remove the closing periods as someone could get confused and accidentally include one of them.